pkg/rpm/Makefile

```diff
@@ -98,7 +98,7 @@ endif
 CONFIGURE_ARGS=\
 	--prefix=/usr \
 	--state=%{_sharedstatedir}/unit \
-	--control="unix:/var/run/control.unit.sock" \
+	--control="unix:/var/run/unit/control.sock" \
```

Comment:
This may raised issue when unit launch outside of systemd (e.g. in docker).
Reverted in next commit